### DISCUSSION TITLE
Référencement dans les projets supportant MQTT Auto-discovery

### DISCUSSION THREAD

**Discussion by Nardol** [MyElectricalData/myelectricaldata_import#297] [Ideas]
Hello,

Il pourrait être intéressant de référencer my electrical data parmi [les projets supportant MQTT Auto-Discovery].
ça donnerait une autre visibilité et ce, dans la documentation officielle.

Si pas le temps de faire la PR, je veux bien la faire :slightly_smiling_face: quoi qu'il en soit je ne veux rien faire de ce type sans accord préalable...

[les projets supportant MQTT Auto-Discovery]: https://www.home-assistant.io/integrations/mqtt/#support-by-third-party-tools

**Comment by pbranly** [comment]
Hello
C’est déjà le cas non ?
Phil

**Reply by Nardol** [reply]
Je viens de parcourir à nouveau la liste des outils référencés et je n'ai trouvé ni enedis2mqtt ni myelectricaldata.

Pour être sûr d'avoir bien formulé ma suggestion : je sais que l'auto-découverte MQTT est supportée, je suggère donc de référencer ce projet dans la documentation de l'intégration MQTT de Home Assistant, dans la partie "Support by third-party tools" qui liste des projets qui proposent le support de l'auto-découverte.

**Comment by pbranly** [comment]
A ok compris . A c’est une décision purement @m4dm4rtig4n .
Phil
